src/platform/plugins/shared/discover/public/application/main/state_management/redux/actions/initialize_single_tab.ts

```diff
@@ -294,9 +297,19 @@ export const initializeSingleTab: InternalStateThunkActionCreator<
     customizationService$.next(customizationService);
 
     // Begin syncing the state and trigger the initial fetch
-    stateContainer.actions.initializeAndSync();
-    stateContainer.actions.fetchData(true);
+    // if this is still the current tab, otherwise mark the
+    // tab to fetch when selected
+    if (getState().tabs.unsafeCurrentId === tabId) {
+      stateContainer.actions.initializeAndSync();
+      stateContainer.actions.fetchData(true);
+    } else {
+      dispatch(
+        internalStateSlice.actions.setForceFetchOnSelect({ tabId, forceFetchOnSelect: true })
+      );
```

Comment:
Ideally we could start fetching even if the user changed tabs, but our fetching logic relies on global services, so using `forceFetchOnSelect` is much lower impact in the meantime.